### PR TITLE
Add code to string worker

### PR DESCRIPTION
**Pull request by ildarsafin**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,12 +3,18 @@ source 'https://rubygems.org'
...
 gem 'rails', '4.0.2'
+gem 'rb-readline', require: false
 
 gem 'pg'
 
 gem 'mail_view'
 gem 'decent_exposure'
 
+group :development do
+  gem 'pry-rails'
```
Comment (dmitrytrager) [inline Gemfile]: we already have `jazz_hands`

File: spec/classes/implicit_string_worker_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
```
Comment (dmitrytrager) [inline spec/classes/implicit_string_worker_spec.rb]: let's rename `explicit` spec and leave other spec as it is

**Comment by dmitrytrager** [comment]
+

File: lib/classes/fs_string_worker.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class FsStringWorker
+  def initialize(input_string)
+    @input_string = input_string
+  end
+
+  def replace
+    longest_word =""
```
Comment (dmitrytrager) [inline lib/classes/fs_string_worker.rb]: can we remove this?

File: spec/classes/explicit_string_worker_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe StringWorker do
+  let(:input_string) { string_generator}
```
Comment (dmitrytrager) [inline spec/classes/explicit_string_worker_spec.rb]: spaces)

File: spec/classes/explicit_string_worker_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+  subject { string_worker }
+
+  context 'it works with random word' do
+    it 'changes the random word' do
+      expect(subject.replace).to eq(fs_string_worker.replace)
+    end
+  end
+
+  def string_generator
```
Comment (dmitrytrager) [inline spec/classes/explicit_string_worker_spec.rb]: can we use 'lorem ipsum' or some kind of?

**Comment by dmitrytrager** [comment]
+

File: spec/classes/explicit_string_worker_spec.rb (hunk 5 above)
Comment (ildarsafin) [inline spec/classes/explicit_string_worker_spec.rb]: We can, but I think that static text will create an opportunity to cheat, won't it?

Comment (dmitrytrager) [inline spec/classes/explicit_string_worker_spec.rb]: Sure. I mean some kind of 'lorem ipsum' generator. [For example](http://lorem.rubyforge.org/)

**Pull request closed without merging**